docs/reference/content/driver-async/tutorials/change-streams.md

```diff
@@ -74,14 +74,40 @@ For additional information on connecting to MongoDB, see [Connect to MongoDB]({{
 
 ## Watch the collection
 
-To create a change stream use the the [`MongoCollection.watch()`]({{<apiref "com/mongodb/client/MongoCollection.html#watch">}}) method.
+To create a change stream use one the [`MongoCollection.watch()`]({{<apiref "com/mongodb/async/client/MongoCollection.html#watch">}}) 
+methods.
 
 In the following example, the change stream prints out all changes it observes.
 
 ```java
 collection.watch().forEach(printBlock, callbackWhenFinished);
 ```
 
+## Watch the database
+
+New in the 3.8 driver and MongoDB 4.0, applications can open a single change stream to watch all non-system collections of a database. To
+create such a change stream use one of the [`MongoDatabase.watch()`]({{<apiref "com/mongodb/async/client/MongoDatabase.html#watch">}}) 
+methods.
+
+In the following example, the change stream prints out all the changes it observes on the given database.
+
+```java
+database.watch().forEach(printBlock, callbackWhenFinished);
+```
+
+## Watch the deployment
```

Comment:
Can we change the deployment to all databases? It follows through to  your explanation below